projects/packages/forms/src/blocks/input-range/edit.js

```diff
@@ -135,6 +141,30 @@ export default function SliderInputEdit( props ) {
 					onKeyDown={ onKeyDown }
 				/>
 			</div>
+			<div className="jetpack-field-slider__text-labels">
+				<input
+					className="jetpack-field-slider__min-text-label"
+					type="text"
+					size={ Math.max( 3, localMinLabel.length ) }
+					value={ localMinLabel }
+					placeholder={ __( 'Add label…', 'jetpack-forms' ) }
+					onChange={ e => setLocalMinLabel( e.target.value ) }
+					onBlur={ () => {
+						onChangeMinLabel?.( localMinLabel );
+					} }
+				/>
+				<input
+					className="jetpack-field-slider__max-text-label"
```

Comment:
```suggestion
					className="jetpack-field-slider__max-text-label"
					autocomplete="off"
```

Sometimes browsers suggest all sorts of things for inputs, this would clean the suggestions out.